projects/utils-ogc/src/lib/wms/wms.service.ts.bak

```diff
@@ -0,0 +1,132 @@
+import { Injectable } from '@angular/core';
```

Comment:
Please remove this file, with git we don't need **.bak**

Reply:
Oh, that's a stupid oversight! Removed.